packages/notification-center/src/i18n/languages/uz.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const UZ: ITranslationEntry = {
+  translations: {
+    notifications: 'bildirishnomalar',
+    markAllAsRead: 'Hammasini o'qing deb belgilang',
+    poweredBy: 'Tomonidan qo'llab-quvvatlanadi',
+    settings: 'sozlamalar',
```

Comment:
```suggestion
    notifications: 'Bildirishnomalar',
    markAllAsRead: 'Hammasini o'qing deb belgilang',
    poweredBy: 'Tomonidan qo'llab-quvvatlanadi',
    settings: 'Sozlamalar',
```
Please capitalise settings and notifications translations.

Reply:
@p-fernandez Thanks I have updated the changes!